opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/entrypoint/HbciResultBodyExtractor.java

```diff
@@ -70,7 +78,34 @@ public PaymentInfoBody extractPaymentInfoBody(ProcessResponse result) {
                 target = "resourceId"
         )
         @Mapping(source = "accountNumber", target = "bban")
+        @Mapping(source = "balances", target = "balances", qualifiedByName = "hbciBalancesToProtocolBalances")
         AccountListDetailBody map(BankAccount account);
+
+
+        @Named("hbciBalancesToProtocolBalances")
+        static List<Balance> hbciBalancesToProtocolBalances(BalancesReport hbciBalances) {
+            List<Balance> protocolBalances = new ArrayList<>();
+            if (hbciBalances != null) {
+                add(protocolBalances, "available", hbciBalances.getAvailableBalance());
+                add(protocolBalances, "credit", hbciBalances.getCreditBalance());
+                add(protocolBalances, "ready", hbciBalances.getReadyBalance());
+                add(protocolBalances, "unready", hbciBalances.getUnreadyBalance());
+                add(protocolBalances, "used", hbciBalances.getUsedBalance());
+            }
+            return protocolBalances;
+        }
+
+        static void add(List<Balance> protocolBalances, String type, de.adorsys.multibanking.domain.Balance hbciBalance) {
+            if (hbciBalance != null) {
+                protocolBalances.add(Balance.builder()
+                    .balanceType(type)
+                    .lastChangeDateTime(hbciBalance.getDate().atStartOfDay(ZoneId.of("Europe/Berlin")).toOffsetDateTime())
```

Comment:
We should avoid using non UTC date/times at least on our side of responsibility. Please use ZoneId.UTC

Reply:
done